documentation/book/ref-kafka-authorization.adoc

```diff
@@ -5,18 +5,17 @@
 [id='ref-kafka-authorization-{context}']
 = Authorization
 
-Authorization can be configured using the `authorization` property in the `Kafka.spec.kafka` resource.
-When the `authorization` property is missing, no authorization will be enabled.
-When authorization is enabled it will be applied for all enabled xref:assembly-configuring-kafka-broker-listeners-{context}[listeners].
-The authorization method is defined by the `type` field.
+You configure authorization for Kafka brokers using the `authorization` property in the `Kafka.spec.kafka` resource.
+If the `authorization` property is missing, no authorization is enabled.
+When enabled, authorization is applied to all enabled xref:assembly-configuring-kafka-broker-listeners-{context}[listeners].
+The authorization method is defined in the `type` field; only Simple authorization is currently supported. 
 
-Currently, the only supported authorization method is the Simple authorization.
+You can optionally designate a list of super users in the `superUsers` field.
 
 == Simple authorization
 
-Simple authorization is using the `SimpleAclAuthorizer` plugin.
-`SimpleAclAuthorizer` is the default authorization plugin which is part of Apache Kafka.
-To enable simple authorization, the `type` field should be set to `simple`.
+Simple authorization in {ProductName} uses the `SimpleAclAuthorizer` plugin, the default Access Control Lists (ACLs) authorization plugin provided with Apache Kafka. ACLs allow you to define which users have access to which resources at a granular level.
+To enable simple authorization, set the `type` field to `simple`.
```

Comment:
@laidan6000 should this instance of simple be capitalized?

Reply:
I'm not sure. What do you think, Paolo? @ppatierno